docs/ldml/tr35-dates.md

```diff
@@ -1707,12 +1708,18 @@ Note: A generic location format is constructed by a part of time zone ID represe
 * "UTC-3" (short)
 * "Гринуич+03:30" (long)
 
-Otherwise (when the offset from GMT is zero, referring to GMT itself) the style specified by the `<gmtZeroFormat>` element is used:
+Otherwise, when the offset from GMT is zero, the style specified by the `<gmtZeroFormat>` element is used:
 
 * "GMT"
 * "UTC"
 * "Гринуич"
 
+Otherwise, when the offset from GMT is not known, the style specified by the `<gmtUnknownFormat>` element is used:
```

Comment:
This sequence doesn't terminate; that is, there should be an unconditional final state

```suggestion
Otherwise (that is, when the offset from GMT is not known), the style specified by the `<gmtUnknownFormat>` element is used:
```